wearApp/src/main/java/dev/johnoreilly/starwars/wearApp/MainActivity.kt

```diff
@@ -49,6 +43,7 @@ sealed class Screen(val title: String) {
     object Lists : Screen("Lists")
 }
 
+@OptIn(ExperimentalWearMaterialApi::class)
```

Comment:
Was trying to avoid this with the opt-in in build.gradle.  Not sure why it's not working in AndroidStudio? Maybe 7.0.4 vs 7.1.0-beta?

But builds work.

Reply:
Yeah, I wasn't sure either why I was still getting error for that here.....using Android Studio Arctic Fox fwiw